DoctorWho.Web/Controllers/DoctorsController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using DoctorWho.Db.Repositories;
+using DoctorWho.Web.Dtos;
+
+namespace DoctorWho.Web.Controllers
+{
+    [Route("api/doctors")]
+    [ApiController]
+    public class DoctorsController : ControllerBase
+    {
+        private readonly IMapper _mapper;
+
+        public DoctorsController(IMapper mapper)
+        {
+            _mapper = mapper;
+        }
+
+        [HttpGet]
+        public IActionResult GetDoctors()
+        {
+            var doctors = DoctorsRepository.current.GetAllDoctors();
```

Comment:
as we said, the repo should be injected into the controller, not as a static field in the repo itself, and that applies to wherever you use that approach here